src/main/java/com/cloudogu/smeagol/wiki/domain/Page.java

```diff
@@ -0,0 +1,65 @@
+package com.cloudogu.smeagol.wiki.domain;
+
+import javax.swing.text.html.Option;
+import java.util.Objects;
+import java.util.Optional;
+
+/**
+ * The page aggregate represents a single wiki page.
+ */
+public class Page {
+
+    private WikiId wikiId;
+    private Path path;
+    private Content content;
+    private Optional<Commit> commit;
+
+    public Page(WikiId wikiId, Path path, Content content) {
+        this.wikiId = wikiId;
+        this.path = path;
+        this.content = content;
+    }
+
+    public Page(WikiId wikiId, Path path, Content content, Commit commit) {
+        this.wikiId = wikiId;
+        this.path = path;
+        this.content = content;
+        this.commit = Optional.of(commit);
+    }
+
+    public void edit(Commit commit, Content content) {
+        this.commit = Optional.of(commit);
+        this.content = content;
+    }
+
+    public WikiId getWikiId() {
+        return wikiId;
+    }
+
+    public Path getPath() {
+        return path;
+    }
+
+    public Content getContent() {
+        return content;
+    }
+
+    public Optional<Commit> getCommit() {
+        return commit;
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') At most one statement is allowed per line, but 2 statements were found on this line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://ecosystem.cloudogu.com/sonar/coding_rules#rule_key=squid%3AS00122)